### ISSUE TITLE
License image in tagged pdf

### ISSUE THREAD

**Issue by henningpohl** [borisveytsman/acmart#612]
Right now the inserted creative commons badge image does not have alt text:
https://github.com/borisveytsman/acmart/blob/97ff970c216f2b7e944c2291a85df99ddd111910/acmart.dtx#L6409

The generated pdf hence just uses the filename as alt text:
<img width="523" height="252" alt="Image" src="https://github.com/user-attachments/assets/cf85d902-3628-4f72-8214-59746f338845" />

I'm not actually sure whether this should be tagged as a figure or an artifact. I'm guessing artifact, but I can also see an argument for figure. Either way, that line should be either
```latex
\href{\ACM@CC@Url}{\includegraphics[height=5ex,alt={Badge for the Creative Commons Attribution 4.0 license}]{doclicense-CC-\ACM@cc@type-88x31}}
```
or
```latex
\href{\ACM@CC@Url}{\includegraphics[height=5ex,artifact]{doclicense-CC-\ACM@cc@type-88x31}}
```

**Comment by Carminiumn** [comment]
I think it should be an artifact, since the text below it has already conveyed its meaning:

> This work is licensed under a Creative Commons Attribution 4.0 International License.
